docs/deployment/deployK8s-AWS.md

```diff
@@ -5,343 +5,158 @@ title: Prepare AWS EKS for Multi-Region Deployments
 
-         > The VPC peering connection status should now show as `Active`.
+    * Click **Create VPC**.
 
-5. Get the subnet information for each cluster node.
+    > Make note of the `VpcId` and `IPv4 CIDR` values for the `eu-west-1` and `us-east-1` VPCs for use in subsequent steps.
 
-      Each cluster node uses a different subnet, so there are three subnets assigned to each VPC. The information displayed contains the subnet ID for each subnet. Use the subnet IDs in the subsequent step to get the associated routing tables.
+    * Repeat this step in `us-east-1` region.
 
-      * At the top right of the page, change the region to **Oregon**.
+2.  Create the transit gateway for each region your deployment is being hosted on. Toggle to the `eu-west-1` region.
 
-      * Go to the **EC2** service, and select **Instances**. Apply a filter, if needed, to find your nodes for the cluster.
+    * Navigate to the **Transit gateways** section and click **Create transit gateway**.
 
-      * Select each node and record the **Subnet ID** for use in a subsequent step.
+    * Add a name tag such as `demo-tgw-eu-west-1`.
 
-      * At the top right of the page, change the region to **Canada Central**, and repeat these steps to find and record the subnet IDs for this VPC.
+    * Add a unique Amazon side Autonomous System Number for each region (ex. 64512 or 64513).
 
-6. Get the routing table associated with the subnets for each VPC.
+    * Disable both the `Default route table association` and `Default route table propagation`.
+    
+    > Make Note if you Enable the options above the default association route table and propagation route table will be created. This may not suit your more complex routing needs, so below you can see how to manually set the associations/propagation route tables.
 
-      * Go to the **VPC** service for the Canada Central region.
+    * Click **Create transit gateway**.
 
-      * In the VPC Dashboard, select **Subnets**.
+    * Repeat this step in `us-east-1` region.
 
-      * For each subnet displayed, record the **Routing Table** value. You might have a single routing table for all of your subnets. You will use the routing table ID or IDs in a subsequent step.
+3.  Create the transit gateway peering connection attachment. Toggle to the `eu-west-1` region.
 
-      * At the top right of the page, change the region to **Oregon**, and repeat these steps to find and record the routing table ID or IDs for this VPC.
+    * Navigate to the **Transit gateway attachments** section and click **Create transit gateway attachments**.
 
-7. Modify the routing table or tables for each VPC to add a route to the other VPC using the peering connection that was created.
+    * Add a name tag such as `demo-peering-attachment-us-east-1`.
 
-      * In the VPC Dashboard, select **Route Tables** for the Oregon region.
+        > Make note that this name is refering to the region it is peering to, not the region it is being create in.
```

Comment:
**_Note: This name refers to the region to which it is peering, not the region in which it is being created._**